src/main/java/com/cgdpd/library/service/AuthorService.java

```diff
@@ -26,4 +29,11 @@ public AuthorDTO createAuthor(String authorName) {
     public boolean authorExist(AuthorId authorId) {
         return authorRepository.existsById(authorId.value());
     }
+
+    @Transactional(readOnly = true)
```

Comment:
Why does this have @Transactional(readOnly = true) and the other doesn't?

Besides it's only required pagination for the book search for now